### PR TITLE
Update branding to 7.0.1

### PR DESCRIPTION
**Pull request by vseanreesermsft**

### CODE REVIEW

**Comment by ViktorHofer** [comment]
I don't know the specifics, but I assume that we don't want to merge this before we release 7.0?

**Comment by carlossanlop** [comment]
> I don't know the specifics, but I assume that we don't want to merge this before we release 7.0?

Confirmed with Matt that we're ok merging this much earlier than usual, it was an explicit request. Also, he mentioned we normally put branding PRs out the week before release (mentioning it here for future reference).

**Comment by lewing** [comment]
@radical please take a look at the wbt failures.

**Comment by radical** [comment]
This needs two fixes:
1. https://github.com/dotnet/runtime/pull/76939, and that needs ..
2. **SDK** fix https://github.com/dotnet/sdk/pull/28627, and it will be usable when the fix shows up in a new 7.0 *installer*.

But ..

3. the installer will be getting the fix from (2) in its `release/7.0.2xx` branch, IIUC
    - for which https://github.com/dotnet/installer/pull/14679 is open, and will bump to SDK's `release/7.0.2xx` branch
4. but the **SDK** PR to get the fix into `release/7.0.2xx` is still open - https://github.com/dotnet/sdk/pull/28668

So:
- (4) needs to merge
- then (3) needs to be updated, and merged
- then wait for a new installer (will that be on the same 7.0 channel for dotnet-install script?), and merge (1)
- Then this PR should be green

🤞

File: eng/Versions.props [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    <ProductVersion>7.0.0</ProductVersion>
+    <ProductVersion>7.0.1</ProductVersion>
     <!-- File version numbers -->
     <MajorVersion>7</MajorVersion>
     <MinorVersion>0</MinorVersion>
-    <PatchVersion>0</PatchVersion>
+    <PatchVersion>1</PatchVersion>
     <SdkBandVersion>7.0.100</SdkBandVersion>
     <PackageVersionNet6>6.0.11</PackageVersionNet6>
     <PreReleaseVersionLabel>rtm</PreReleaseVersionLabel>
```
Comment (carlossanlop) [inline eng/Versions.props:11]: Per @dougbu 's note in chat, changing this to servicing:
```suggestion
    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
```

**Comment by radical** [comment]
update: I'm disabling the failing tests in https://github.com/dotnet/runtime/pull/76939 , that way we won't have to wait to get a new installer build with the sdk fix, since that take some time to be available.

**Comment by carlossanlop** [comment]
> update: I'm disabling the failing tests in #76939 , that way we won't have to wait to get a new installer build with the sdk fix, since that take some time to be available.

So does that mean I can consider this PR is unblocked and merge it?

**Comment by radical** [comment]
> > update: I'm disabling the failing tests in #76939 , that way we won't have to wait to get a new installer build with the sdk fix, since that take some time to be available.
> 
> So does that mean I can consider this PR is unblocked and merge it?

No, it still has some fixes that are needed here. So, once we merge that, this will be unblocked. The difference from earlier being that we won't have to wait to get a new installer build with the fix.

**Comment by carlossanlop** [comment]
Merging as the CI failures look unrelated, and the timeouts are due to ongoing helix problems. We'll be monitoring the rollout builds.

**Pull request merged**